TAScheduler/Managment/UserManagement.py

```diff
@@ -1,3 +1,8 @@
+from django.db import models
+import TAScheduler.models
```

Comment:
```suggestion
from TAScheduler.models import UserProfile
```

Reply:
The three lines mentioning models here can be replaced with just one as outlined above.